cookiecutter/tap-template/{{cookiecutter.tap_id}}/pyproject.toml

```diff
@@ -18,15 +18,19 @@ keywords = [
 ]
 classifiers = [
     "Intended Audience :: Developers",
+{%- if cookiecutter.license == "Apache-2.0" %}
     "License :: OSI Approved :: Apache Software License",
+{%- endif %}
     "Operating System :: OS Independent",
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3.12",
     "Programming Language :: Python :: 3.13",
 ]
-license = "Apache-2.0"
+{%- if cookiecutter.license != "None" %}
+license = "{{cookiecutter.license}}"
+{%- endif %}
 license-files = [ "LICENSE" ]
```

Comment:
(let's start a thread)

> Let me know how you feel about [7b67928](https://github.com/meltano/sdk/commit/7b679287f41a7fc6f8260a4da5c88acc94842f96).

@ReubenFrankel was the metadata validation failing before that commit?

Reply:
I didn't check. I just felt like it would be fine to always include given that a user may end up adding a `LICENSE` file later on.

Reply:
That makes sense 👍